### DISCUSSION TITLE
Submodule update remote

### DISCUSSION THREAD

**Discussion by sarahamsaw** [gitpython-developers/GitPython#1536] [Q&A] [answered]
From reading the documentation, I assumed that 
`git submodule update --remote`
could be recreated in GitPython by the following
`repo.submodule_update(to_latest_revision=True, recursive=False)`. 

If I print out the progress, it looks like the latter one does a fetch, but doesn't update the submodule. Is there a way to do this in GitPython besides just calling `repo.git.submodule('update', '--remote')`?

Thanks!

**Comment by Byron** [comment, marked as answer]
I actually don't know (anymore) and would recommend to check the test suite via `test_submodule.py` or the likes - maybe it has invocations that show how to do certain things.

If in doubt, it can be assumed that it's not working, generally I wouldn't recommend using this submodule implementation as it's non-standard, tries to be smart and helpful, but due to lack of maintenance it might do the opposite by now.
